depends/packages/native_ccache.mk

```diff
@@ -1,8 +1,8 @@
 package=native_ccache
-$(package)_version=3.3.1
-$(package)_download_path=https://www.samba.org/ftp/ccache
-$(package)_file_name=ccache-$($(package)_version).tar.bz2
-$(package)_sha256_hash=cb6e4bafbb19ba0a2ec43386b123a5f92a20e1e3384c071d5d13e0cb3c84bf73
+$(package)_version=3.7.11
```

Comment:
Aaand in the week since I prepared this update, `ccache 3.7.12` was released 😂 I'll make a follow-up PR with that update.